nodejs_config/config.js

```diff
@@ -3,5 +3,7 @@
  */
 exports.App = {
     CouchServerIp: 'http://127.0.0.1:5984',
-    serverIp: 'http://localhost:3000'
+    serverIp: 'http://localhost:3000',
+    mailGunApiKey: 'key-9a6d6bd3a96a3315693cdc72dc651a43',
```

Comment:
we need to take mailgun key out of public repo too. and also use our account in a way that we can utilise their 10,000 free emails wala package.